### ISSUE TITLE
security concerns with authentication

### ISSUE THREAD

**Issue by SwapnilSoni1999** [tomquirk/linkedin-api#172]
when we log in in with email and password, And whenever if we visit linkedin.com on browser then it asks to reset password by asking to confirm identity. So seems quite risky for the particular account What else can be done to prevent such issues

**Comment by PMLS3** [comment]
Same issue here, when I deploy got app engine, then I get the server error and linkedin email to with security concerns. Running and working on it locally is working 100

**Comment by australianresearchgroup** [comment]
Same issue here.

The python script is throwing the following error;

line 157, in _do_authentication_request
    raise ChallengeException(data["login_result"])
linkedin_api.client.ChallengeException: CHALLENGE

then refresh the linkedin page and it boots me out to a password reset

**Comment by abinpaul1** [comment]
A sort of workaround is discussed in #109 .

**Comment by SwapnilSoni1999** [comment]
> A sort of workaround is discussed in #109 .

Interesting but the thing is that module is based on selenium 
Still if we can inject those cookies with this commandline tool then it would be pretty good

**Comment by abinpaul1** [comment]
That's true. Some work was actually done on this and mentioned in the same thread #109 . 

Also, the workaround I was talking about refers to https://github.com/tomquirk/linkedin-api/issues/109#issuecomment-853808258 . It
 doesn't seem to make use of selenium and it's something that had to be run just once as well.
